src/main/kotlin/org/assertj/generator/gradle/internal/tasks/AssertionsGeneratorReport.kt

```diff
@@ -0,0 +1,179 @@
+/*
+ * Copyright 2023. assertj-generator-gradle-plugin contributors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package org.assertj.generator.gradle.internal.tasks
+
+import com.google.common.reflect.TypeToken
+import org.assertj.assertions.generator.AssertionsEntryPointType
+import java.io.File
+import java.util.TreeSet
+
+internal class AssertionsGeneratorReport(
+  private val directoryPathWhereAssertionFilesAreGenerated: File,
+  private val inputClasses: Collection<String>,
+  private val excludedClassesFromAssertionGeneration: Collection<TypeToken<*>>,

```

Comment:
Making these constructor parameters and computing `inputPackages` was the only change that was not a straight `Konvert`.